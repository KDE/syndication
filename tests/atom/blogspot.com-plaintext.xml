<?xml version='1.0' encoding='UTF-8'?>
<?xml-stylesheet href="http://www.blogger.com/styles/atom.css" type="text/css"?>
<!-- blogspot.com feed from 2006/10
     * heavy usage of <link>
     * opensearch extension
     * using plain text, whitespace preservation seems intended  but "Atom Processors MAY collapse white space"
       and we do it
-->
<feed xmlns='http://www.w3.org/2005/Atom' xmlns:openSearch='http://a9.com/-/spec/opensearchrss/1.0/'>
<id>http://afreytes.blogspot.com/feeds/posts/default</id>
<updated>2006-10-30T20:53:52.240-04:00</updated>
<title type='text'>The Information Soldier</title>
<link rel='alternate' type='text/html' href='http://afreytes.blogspot.com/index.html'></link>
<link rel='http://schemas.google.com/g/2005#feed' type='application/atom+xml' href='http://afreytes.blogspot.com/feeds/posts/default'></link>
<link rel='http://schemas.google.com/g/2005#post' type='application/atom+xml' href='http://afreytes.blogspot.com/feeds/posts/default'></link>
<link rel='self' type='application/atom+xml' href='http://afreytes.blogspot.com/feeds/posts/default?max-results=25'></link>
<link rel='next' type='application/atom+xml' href='http://afreytes.blogspot.com/feeds/posts/default?start-index=26&amp;max-results=25'></link>
<author><name>afreytes</name><uri>http://afreytes.blogspot.com</uri><email>afreytes@gmail.com</email></author>
<generator version='7.00' uri='http://beta.blogger.com'>Blogger</generator>
<openSearch:totalResults>244</openSearch:totalResults>
<openSearch:startIndex>1</openSearch:startIndex>
<openSearch:itemsPerPage>25</openSearch:itemsPerPage><

entry>
<id>http://afreytes.blogspot.com/feeds/posts/default/8213616807784621006</id>
<published>2006-10-30T20:41:00.000-04:00</published>
<updated>2006-10-30T20:53:51.901-04:00</updated>
<title type='text'>La Cancion De La Semana - Ninja Gaiden Medley</title>
<summary type='text'>Primero algo para refrescarle la memoria a los nerdogeeks que me leen:



Y ahora la canción de la semana; un Medley De Ninja Gaiden cortesía de los Minibosses:



Los Minibosses han sido exitosos en capturar la nostalgía que sentimos los gamers jarcorosos que recordamos cuando los videojuegos tenían 32 colores* en vez de 32 millones y audio mono en vez de Dolby 5.1.

* (Nota aclaratoria: </summary>
<link rel='alternate' type='text/html' href='http://afreytes.blogspot.com/2006/10/la-cancion-de-la-semana-ninja-gaiden.html'></link>
<link rel='self' type='application/atom+xml' href='http://afreytes.blogspot.com/feeds/posts/default/8213616807784621006'></link>
<link rel='edit' type='application/atom+xml' href='http://afreytes.blogspot.com/feeds/posts/default/8213616807784621006'></link>
<author><name>afreytes</name><uri>http://afreytes.blogspot.com</uri><email>afreytes@gmail.com</email></author>
</entry></feed>
