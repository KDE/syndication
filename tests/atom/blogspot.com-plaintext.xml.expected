# Feed begin ######################
title: #The Information Soldier#
link: #http://afreytes.blogspot.com/index.html#
# Person begin ####################
name: #afreytes#
uri: #http://afreytes.blogspot.com#
email: #afreytes@gmail.com#
# Person end ######################
# Item begin ######################
id: #http://afreytes.blogspot.com/feeds/posts/default/8213616807784621006#
title: #La Cancion De La Semana - Ninja Gaiden Medley#
link: #http://afreytes.blogspot.com/2006/10/la-cancion-de-la-semana-ninja-gaiden.html#
description: #Primero algo para refrescarle la memoria a los nerdogeeks que me leen:



Y ahora la canción de la semana; un Medley De Ninja Gaiden cortesía de los Minibosses:



Los Minibosses han sido exitosos en capturar la nostalgía que sentimos los gamers jarcorosos que recordamos cuando los videojuegos tenían 32 colores* en vez de 32 millones y audio mono en vez de Dolby 5.1.

* (Nota aclaratoria:#
datePublished: #Tue Oct 31 01:41:00 2006#
dateUpdated: #Tue Oct 31 01:53:51 2006#
# Person begin ####################
name: #afreytes#
uri: #http://afreytes.blogspot.com#
email: #afreytes@gmail.com#
# Person end ######################
# Item end ########################
# Feed end ########################

